### PR TITLE
DFA-2246-Roadmap update 

### PR DESCRIPTION
**Pull request by bethwalton**
## What 

Updating the content on the roadmap to add more updates, and to clarify which workstream they relate to. 

## Why 

Our product offer is expanding and we're onboarding services to identity.

### CODE REVIEW

**Review by CharlesIC — CHANGES_REQUESTED** [review]
Please revert changes to this file

File: package-lock.json [hunk 1]
```diff
@@ -2734,6 +2734,7 @@
       "version": "2.3.2",
       "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-2.3.2.tgz",
       "integrity": "sha512-xiqMQR4xAeHTuB9uWm+fFRcIOgKBMiOBP+eXiyT7jsgVCq1bkVygt00oASowB7EdtpOHaaPgKt812P9ab+DDKA==",
+      "dev": true,
```
Comment (CharlesIC) [inline package-lock.json:2737]: Please revert changes to this file

**Review by CharlesIC — CHANGES_REQUESTED** [review]

**Review by CharlesIC — COMMENTED** [review]
If possible, please squash all changes into one commit (don't worry if you're not sure how to do it, or feel free to ask for help!)

File: package-lock.json (hunk 1 above)
Comment (bethwalton) [reply package-lock.json]: Done

**Review by CharlesIC — APPROVED** [review]
Some formatting issues (`p` tags are indented too much) but the HTML is correct.
I've not reviewed the content itself.

**Review by JohnWattsGDS — APPROVED** [review]

**Review by JohnWattsGDS — APPROVED** [review]

**Pull request merged**